samples/net/openthread/coap_client/src/board_utils.c

```diff
@@ -0,0 +1,56 @@
+#include "board_utils.h"
+
+#include <device.h>
+
+static void init_button(const char *name, u32_t pin,
```

Comment:
Don't we have such a high level BSP layer in NCS? Or each example itself need to define Led and Button handling?

Reply:
There's https://github.com/lmaciejonczyk/fw-nrfconnect-nrf/blob/master/include/dk_buttons_and_leds.h, several samples use it.

Reply:
replaced